docs/basic-features/environment-variables.md

```diff
@@ -4,7 +4,17 @@ description: Learn to add and access environment variables in your Next.js appli
 
 # Environment Variables
 
-> This document is for Next.js versions 9.4 and up. If you’re using an older version of Next.js, upgrade or refer to [Environment Variables in next.config.js](/docs/api-reference/next.config.js/environment-variables.md).
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version   | Changes                                                             |
+| --------- | ------------------------------------------------------------------- |
+| `v12.2.3` | `locale`, `locales`, `defaultLocale`, and `notFound` options added. |
+| `v9.4.0`  | Support `.env` and `NEXT_PUBLIC_` introduced.                       |
+
+This document is for Next.js versions 9.4 and up. If you’re using an older version of Next.js, upgrade or refer to [Environment Variables in next.config.js](/docs/api-reference/next.config.js/environment-variables.md).
```

Comment:
I think this paragraph should be outside `<details>`, right?

Reply:
I've moved it out